swagger/v1/api_v1.json

```diff
@@ -1453,6 +1519,28 @@
           "reading_time_minutes"
         ]
       },
+      "FollowedTag": {
+        "description": "Resprentation of a tag",
```

Comment:
Theres a typo here, however the original description key in `swagger_helper` file has the correct spelling which is strange coz the json file gets generated based on the swagger helper and the spec.